content/working_with/console/customization/overrides.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bt_wiki
+title: Labels Overrides
+category: Console
+draft: false
+weight: 180
+---
+
+You can override a number of texts and labels used in {{< param cfy_console_name >}} by supplying a file containing replacements for default texts or labels.
+
+To do that:
+
+1. Create `/opt/cloudify-stage/dist/userData/overrides.json` file.
+1. Go to https://github.com/cloudify-cosmo/cloudify-stage/blob/<version>-build/app/translations/en.json (replace `<version>` with your product version, for example `5.1.0`), choose configurations parts to modify (you can omit all unmodified parts to use default values) and save them to the created file.
```

Comment:
I dont think it is a good practice to let the user set the version in the URL, and then get the file from git.

Do we have such a file on the manager? can the user copy it from the folder where it resides?

Reply:
We don't have this file on manager, so the user can't just copy it.

Please note that although the link points to github it is a regular HTTP link that can be clicked and opened in a browser.
My initial commit was to link directly to `master` branch without the need for link editing. Should I revert to that? Or maybe just use `5.1.1-build` branch?

Reply:
@ofercloudify That's important question as well for me doing changes in Composer right now (RD-233). Translation file is not exposed in the production code there as well, it is bundled.

If we want to have it on the manager, let us know. 
We would need to add those files to the Stage and Composer package.

Reply:
OK, if this is the case we will stay with github.
I suppose the right approach to the version should be by adding a parameter to the config.toml for the version number and embed it in the link. The downside would be updating it with every release but that is on me.